### DISCUSSION TITLE
Markdown editor suitable for nuxt3？

### DISCUSSION THREAD

**Discussion by facexyy** [nuxt/nuxt#24080] [General]
I have not found a markdown editor npm library that is more suitable for the nuxt3 ecosystem.

Does nuxt3 have a relevant ecological markdown editor?

**Comment by MichaelJCole** [comment]
If you are looking for syntax highlighting, try the Monaco Editor.
Then run it through a markdown -> html renderer and load it into a browser.
I think Vue has a feature to load "unsafe" html into a component.  Markdown is relatively "safe", except where it can include html directly which may or may not be safe depending on your users.

I've looked and haven't found a package that does all of this together.  I think that's because the security and styling are so app specific.

**Reply by facexyy** [reply]
Thanks for your answer, I will go and find out about Monaco Editor
